src/main/scala/io/iohk/ethereum/blockchain/sync/fast/FastSync.scala

```diff
@@ -384,6 +380,7 @@ class FastSync(
 
     private def blockHeadersError(peer: Peer, blacklistReason: BlacklistReason): Unit = {
       blacklist.add(peer.id, blacklistDuration, blacklistReason)
+      log.debug("Blacklisting peer with id {}, reason: {}", peer.id, blacklistReason.toString)
```

Comment:
This should already be logged in https://github.com/input-output-hk/mantis/blob/04208b047c1f9c4917e8fb662709911ab59afe44/src/main/scala/io/iohk/ethereum/blockchain/sync/Blacklist.scala#L152